doc/whats_new/v0.20.rst

```diff
@@ -282,6 +282,10 @@ Classifiers and regressors
 - Fixed a bug in :class:`sklearn.linear_model.Lasso`
   where the coefficient had wrong shape when ``fit_intercept=False``.
   :issue:`10687` by :user:`Martin Hahn <martin-hahn>`.
+  
+- Fixed a bug in :func:`sklearn.linear_model.logistic` where the the 
+  multi_class='multinomial'with binary output was not converging with and without
```

Comment:
Space after quote please.

I'm not sure about the terminology "was not converging". I thought there was an error in the warm_start case. And I wasn't aware of an issue in the warm_start=False case

Reply:
So, basically the warm_start= true wasn't converging with the warm_start=False